pwndbg/pdb.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+This module serves as utility for the developers to get pdb debugging session right, whenever standard:
+    import pdb; pdb.set_trace()
+Can't do its job properly (that happens often as gdb does some magic with stdout/stderr).
+
+Thanks to Dmoreno: http://stackoverflow.com/questions/17074177/how-to-debug-python-cli-that-takes-stdin
+
+Usage:
+    import pwndbg.pdb
+
+Importing the module will print out pretty message on what to do next.
+"""
+import functools
+import pdb
```

Comment:
For Python 3 it makes... any command unusable:
https://asciinema.org/a/0a4h2fe7yfq5d18djtoav3ezr

I have tried to remove binary mode from `pwndbg.stdio` and that's what I got, huh:
```
$ gdb
Loaded 106 commands. Type pwndbg [filter] for a list.
pwndbg> version
Traceback (most recent call last):
  File "/home/dc/installed/pwndbg/pwndbg/commands/__init__.py", line 55, in __call__
    with pwndbg.stdio.stdio:
  File "/home/dc/installed/pwndbg/pwndbg/stdio.py", line 34, in __enter__
    sys.stdin  = get('/dev/stdin', 'r')
  File "/home/dc/installed/pwndbg/pwndbg/stdio.py", line 25, in get
    return io.open(fd, mode, buffering=0)
ValueError: can't have unbuffered text I/O
```